setup.py

```diff
@@ -159,7 +159,7 @@ def build_libscalene(self):
     install_requires=[
         "rich>=9.2.0",
         "cloudpickle>=1.5.0",
-        "numpy",
+#        "numpy",
```

Comment:
Maybe plain delete this?

Reply:
Agreed; this was tentatively commented out.